build/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/2b1476549e.js" crossorigin="anonymous"></script>
+    <title>TO DO LIST</title>
+<script defer src="index.js"></script></head>
+
+<body>
+    <div class="container">
+    </div>
+</body>
+
+</html>
```

Comment:
- [ ] [OPTIONAL] I will recommend that you ensure this code is properly indented in order to follow the ES6 syntax and to ensure good readability. I will recommend that you ensure this code is properly indented in order to follow the ES6 syntax and to ensure good readability.